docs/next/guides/sidebar.js

```diff
@@ -1,189 +1,189 @@
 const gettingStartedItems = [
-  'guides/getting-started/introduction',
-  'guides/getting-started/demo',
-  'guides/getting-started/installation',
-  'guides/getting-started/binding-to-data',
-  'guides/getting-started/saving-data',
-  'guides/getting-started/setting-options',
-  'guides/getting-started/grid-size',
-  'guides/getting-started/events-and-hooks',
-  'guides/getting-started/license-key',
+  { link: 'guides/getting-started/introduction' },
+  { link: 'guides/getting-started/demo' },
+  { link: 'guides/getting-started/installation' },
+  { link: 'guides/getting-started/binding-to-data' },
+  { link: 'guides/getting-started/saving-data' },
+  { link: 'guides/getting-started/setting-options' },
+  { link: 'guides/getting-started/grid-size' },
+  { link: 'guides/getting-started/events-and-hooks' },
+  { link: 'guides/getting-started/license-key' },
```

Comment:
As an example, can you add `guides/integrate-with-react/react-redux-example` in this section, only for React?